### PR TITLE
Show pokemon moves in pokemon list! #86 

### PR DESCRIPTION
**Pull request by mbrainiac**

### CODE REVIEW

File: app/scripts/inventory.js [hunk 1]
```diff
@@ -30,5 +36,9 @@
         return allItems[id];
     }
 
+    service.getMove = function(id) {
+      return allMoves[id + ''];
```
Comment (tplk) [inline app/scripts/inventory.js:40]: Hey, can you tell why would you need single brackets?

Comment (deathart) [inline app/scripts/inventory.js:40]: return allMoves[id];

Comment (tplk) [inline app/scripts/inventory.js:40]: @deathart I don't understand you.

Comment (mbrainiac) [inline app/scripts/inventory.js:40]: It's the same as return allMoves[id];, key of map should be string.

Comment (tplk) [inline app/scripts/inventory.js:40]: @mbrainiac Oh, okay, thanks. That's what I thought.

**Pull request closed without merging**